src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -188,7 +188,7 @@ public FixHandler(IHandlerContext context) {
 
             long time = now.until(scheduleTime, ChronoUnit.SECONDS);
             executorService.scheduleAtFixedRate(() -> {
-                this.close();
+                channel.close();
```

Comment:
Before by calling `close()` we were sending logout, now we're just closing a channel

Reply:
We were also waiting for logout response